homeassistant/components/history.py

```diff
@@ -35,6 +35,49 @@ def last_5_states(entity_id):
     return recorder.query_states(query, (entity_id, ))
 
 
+def get_significant_states(
+        start_time, end_time=None, entity_id=None):
+    """
+    Return states changes during UTC period start_time - end_time.
+    """
+    all_states_significant = ('thermostat',)
+    domain_where = "domain in ({})".format(
+        ",".join(["'%s'" % x for x in all_states_significant]))
+    where = "({} or last_changed=last_updated) AND last_updated > ? ".format(
```

Comment:
Can you merge this and above assignment into 1. You can use multiline strings to help keep the query readable.